website/docs/troubleshooting.md

```diff
@@ -210,11 +210,24 @@ class A
 end
 ```
 
-## Alternatives to `T.unsafe`
+### `T.cast`: a safer alternative to `T.unsafe`
 
 `T.unsafe` is maximally unsafe. It forces Sorbet to forget all type information
 statically---sometimes this is more power than we need. For the cases where we
 the programmer know of an invariant that isn't currently expressed in the type
 system, [`T.cast`](type-assertions#tcast) is a good middle-ground.
 
-<!-- TODO(jez) Document .on_failure / .checked once API is stable. -->
+### Runtime escape hatches
+
+Both `T.unsafe` and `T.cast` are escape hatches from Sorbet's static checks.
```

Comment:
I don't understand the metaphor with an escape hatch.

Reply:
@Morriar does that go for this entire doc? Half of troubleshooting.md is talking about escape hatches.

Reply:
I'm bike shedding here, but if someone is reading this page, he's most likely already lost, and you're making Sorbet sounds like a sinking submarine :p

Earlier in the chapter, it's used as a H2, while intriguing, the title doesn't really help me to decide if I need to read this section. What do you think about something more explanatory **Relaxing type constraints locally**?

Not saying "escape hatch" is a bad term, just that I would not over use it.